developerportal/urls.py

```diff
@@ -28,8 +31,17 @@
 ]
 
 
+# Custom 403 handling, sending either a rate-limited response or a regular Forbidden
+def handler403(request, exception=None):
+    if isinstance(exception, Ratelimited):
+        return rate_limited(request, exception)
+    return permission_denied(request, exception)
+
+
 if settings.DEBUG:
     urlpatterns += [
+        url(r"^429/$", rate_limited, {"exception": Ratelimited()}),
```

Comment:
What are these for? Is it just so you can manually preview, in the browser, what these error pages look like?

Reply:
Yep, exactly! Makes local dev easier and because `DEBUG` is never True in our deployed sites, remains local only